lib/mdl/ruleset.rb

```diff
@@ -31,6 +32,14 @@ def params(params = nil)
       @params.update(params) unless params.nil?
       @params
     end
+
+    def docs(url)
+      @generate_docs = ->(_, _) { url }
```

Comment:
I prefer the more explicit `lambda` over the `->`.

Reply:
Your rubocop file disagrees with you 😅  I'll update the linter too 😊